guide/english/bulma/get-started/index.md

```diff
@@ -56,4 +56,8 @@ For bulma to work correctly, you need to make your webpage responsive.
 ```
 
 ### Bulma-start
-```bulma-start``` is a tiny ```npm``` package that includes the npm dependencies you need to build your own website with Bulma.
\ No newline at end of file
+```bulma-start``` is a tiny ```npm``` package that includes the npm dependencies you need to build your own website with Bulma.
+
+### Further References:
```

Comment:
```suggestion
## Further References
```

I've increased the title's level since it doesn't really fall under the subject of `bulma-start`, but more generally under getting started with Bulma.